mage_integrations/mage_integrations/sources/stripe/__init__.py

```diff
@@ -205,11 +207,7 @@ def get_schema(cls, stream_name):
 
     @classmethod
     def is_selected(cls, stream_name):
-        # TODO (tommy dang): support syncing multiple streams aka sub-streams
         stream = cls.get_catalog_entry(stream_name)
-        if not stream:
```

Comment:
why do you remove this?

Reply:
Added in 028336556fca308242837d69d26186c17481362d